src/index.js

```diff
@@ -27,6 +27,7 @@ export default (opts) => {
   const options = Object.assign(
     {
       optsEslint: null,
+      maxWarnings: -1
```

Comment:
Yeah probably just warns. Just run eslint fix and I think this PR is good to go.